stl/inc/shared_mutex

```diff
@@ -77,6 +91,8 @@ public:
     shared_timed_mutex(const shared_timed_mutex&)            = delete;
     shared_timed_mutex& operator=(const shared_timed_mutex&) = delete;
 
+    _Acquires_shared_lock_(this)

```

Comment:
This should be `_Acquires_exclusive_lock_` (same below for `try_lock`, `try_lock_for`, and `try_lock_until`, as well as `_Releases_exclusive_lock` for `unlock`)

Reply:
Fixed